components/editor/CodeBlock.tsx

```diff
@@ -0,0 +1,583 @@
+              ${showLineNumbers ? 'ml-0' : 'ml-3'} ${textSizeClassName}`}
+            language={codeBlocks &&  codeBlocks[activeBlock].language ? codeBlocks[activeBlock].language : language}
+            style={theme}
+            showLineNumbers={showLineNumbers}
+            startingLineNumber={startingLineNumber}
+            lineNumberContainerProps={{
+              className: 'pl-2 float-left left-0 sticky bg-code-editor-dark',
+              style: {}
+            }}
+            lineNumberProps={(lineNumber: number) => {
+              const isHighlighted = highlightedLines && highlightedLines.includes(lineNumber);
+
+              return {
+                className: `${isHighlighted ? 'bg-code-editor-dark-highlight text-gray-500' : 'text-gray-600'}
+                block pl-2 pr-2`
+              };
+            }}
+            wrapLines={true}
+            lineProps={(lineNumber: number) => {
+              const isHighlighted = highlightedLines && highlightedLines.includes(lineNumber);
+
+              return {
+                className: `${isHighlighted ? 'bg-code-editor-dark-highlight block ml-10 w-full' : ''} pr-8`
+              };
+            }}
+            codeTagProps={{
+              className: 'mr-8'
+            }}
+          >
+            {codeBlocks ? [codeBlocks[activeBlock].code] : ''}
+          </Highlight>
+        </div>
+      </div>
+    );
+  }
+
+  return (
+    <>
+      <div className={`relative z-10 my-8 max-w-full overflow-auto rounded bg-code-editor-dark pt-2 ${className}`}>
+        {hasWindow && (
+          <div className='pb-2 pl-4'>
+            <span className='mr-2 inline-block size-2.5 rounded-full bg-mac-window-close'></span>
+            <span className='mr-2 inline-block size-2.5 rounded-full bg-mac-window-minimize'></span>
+            <span className='mr-2 inline-block size-2.5 rounded-full bg-mac-window-maximize'></span>
+          </div>
+        )}
+        {
+          showCopy && (
+            <div className='z-10'>
+              <button
+                onClick={onClickCopy}
+                className='absolute right-2 top-1 z-50 cursor-pointer bg-code-editor-dark text-xs
+                text-gray-500 hover:text-gray-300 focus:outline-none'
+                title='Copy to clipboard'
+                data-test='copy-button'
+              >
+                {
+                  showIsCopied &&
+                  <span className='mr-2 inline-block pl-2 pt-1' data-testid='clicked-text'>Copied!</span>
+                }
+                <span className='inline-block pt-1'><IconClipboard className='-mt-0.5 inline-block size-4' /></span>
+              </button>
+            </div>
+          )
+        }
+        {renderHighlight()}
+      </div>
+      {showCaption && caption && (
+        <Caption>{caption}</Caption>
+      )}
+    </>
+  );
+};
+
+lowlight.registerLanguage('asyncapi+yaml', (hljs : any) => {
+  const LITERALS: Literal[] = ['true', 'false', 'yes', 'no', 'null'];
```

Comment:
All these constants should be in separate file and they should be imported from that constant file. You can create a file `EditorLanguageConstants.ts` to store all constants.

Reply:
This code was removed on removal of lowlight.